about.html

```diff
@@ -117,6 +144,40 @@ <h2 class="accordion-header">
                 <div class="accordion-body">
                   The redesign will not impede emergency vehicles. All DOT changes are approved by the FDNY. The new bike lanes will be accessible to emergency vehicles should they need it, effectively allowing them to bypass any traffic on the boulevard (an improvement from the current design).
                   <p>Although there are some signs on McGuinness indicating it's an evacuation route, specific evacuation routes are no longer in use by NYC. Should an evacuation be necessary, residents will still be able to use the boulevard on foot, by vehicle, or by bicycle.</p>
+                  <p>In Paris, new cycle paths have <a href="https://www-leparisien-fr.translate.goog/amp/paris-75/a-paris-le-delai-dintervention-des-pompiers-a-raccourci-sur-une-urgence-une-minute-ca-joue-02-03-2023-D3MEX74B4NCKHCYROAQBECNVGE.php?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp">significantly reduced</a> emergency vehicle response time.</p>
+                </div>
+              </div>
+            </div>
+            <div class="accordion-item">
+              <h2 class="accordion-header">
+                <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
+                  Businesses are complaining about the lack of outreach. How has DOT consulted the community on the plans?
```

Comment:
How about we be a bit more neutral here haha? Maybes something like "I've heard there's been a lack of outreach. How has DOT consulted the community on the plan?"